website/docs/externaldata.md

```diff
@@ -498,21 +532,21 @@ To ensure a request to the Gatekeeper webhook is coming from the API server, Gat
     apiVersion: v1
     clusters:
     - cluster:
-        certificate-authority-data: <clientca.crt> # same value as provided in gatekeeper webhook secret's clientca.crt 
+        certificate-authority-data: <ca-crt> # same value as provided in gatekeeper webhook secret's clientca.crt 
         server: https://<gatekeeper-webhook-service-name>.<gatekeeper-namespace>.svc:443
-      name: kind-kind
+      name: <cluster-name>
     contexts:
     - context:
-        cluster: kind-kind
+        cluster: <cluster-name>
         user: api-server
-      name: kind-kind
-    current-context: kind-kind
+      name: <context-name>
+    current-context: <context-name>
     kind: Config
     users:
     - name: api-server
       user:
-        client-certificate-data: <client-certificate-signed-with-ca-authority> # i.e. cert signed with apiserver's ca
-        client-key-data: <key-mathcing-the-signed-client-certificate>
+        client-certificate-data: <apiserver-client.crt> 
```

Comment:
nit: you are using `apiserver-client.crt` here but `ca-crt` and `clientca.crt` else where. perhaps make them consistent to reduce confusion.

Reply:
Done! updated the doc to make it consistent.